docs/configuration/OVERVIEW.md

```diff
@@ -266,102 +500,101 @@ export class AppComponent implements OnInit, OnDestroy {
   private getStaticConfig() {
     if (this.configurationService) {
       this.configurationService.getConfigFromBodyTag('staticconfig');
     }
   }
-
-...
 }
 ````
 
-#### Handle global config
+#### Handle global configuration
 
 ````typescript
 import {AREA_1_CONFIG_DEFAULT} from '@scope/area-1';
 import {AREA_2_CONFIG_DEFAULT} from '@scope/area-2';
 import {ConfigurationBaseService} from '@o3r/configuration';
 
-...
-
 export class AppComponent implements OnInit, OnDestroy {
 
-  constructor(...
-              @Optional() private configurationService?: ConfigurationBaseService) {
+  constructor(
+    @Optional() private configurationService?: ConfigurationBaseService
+  ) {
     this.registerDefaultGlobalConfig();
   }
 
-  // As mentioned above, we have to merge common configs coming from libraries and to pass the result as global config to the store
-  // The config service has a method which is doing this
+  // As mentioned above, we have to merge the common configurations coming from the libraries and use the result as a global configuration
+  // The configuration service has a method that does this
   private registerDefaultGlobalConfig() {
-    // Compute the global config based on all common configuration from libraries
+    // Compute the global configuration based on all common configurations from the libraries
     const global = {...AREA_1_CONFIG_DEFAULT, ...AREA_2_CONFIG_DEFAULT};
     this.configurationService.extendConfiguration(global);
   }
-
-...
 }
-
 ````
 
+## Naming convention
+
+| Attribute                   | Pattern     |
+| --------------------------- | ----------- |
+| **Configuration file name** | *.config.ts |
+| **Configuration name**      | *Config     |
+
 ## Key takeaways
 
-The configuration mechanism is based on __Store Configuration__ and __Cms Adapters__.
+The configuration mechanism is based on the __Configuration Service__ and __Extractors__.
 
-__Default configuration__ for a component type is the one taken from the .config.ts file associated to that component.
+The __Default configuration__ for a component is the one taken from the `.config.ts` file associated to that component.
 
-- The config metadata is extracted and computed from the .config.ts file of each component
+- The configuration metadata is extracted and computed from the `.config.ts` file of each component.
 
-__Static configuration__ is the one injected in the index.html of the application.
+The __Static configuration__ is the one injected in the `index.html` of the application.
 
-- It's the configuration which is modified inside the CMS or by hand (implementation teams)
-  - It will be injected as a data attribute on the body tag (data-staticconfig). At bootstrap, the value from the data
-    attribute will be read and the store updated.
-- It will override the default configuration
+- The configuration is modified inside the CMS or by hand by implementation teams for example.
```

Comment:
Same remark as before: we can't really say "the CMS" as if we were providing one